tests/cypress/e2e/menuActions/areaActions.cy.ts

```diff
@@ -35,7 +35,8 @@ describe('Area actions', () => {
         menu.shouldHaveItem('Edit');
     });
 
-    it('Checks that content can be pasted into the area', () => {
+    // FIXME issue with right click probably due to empty content with new centered buttons
+    it.skip('Checks that content can be pasted into the area', () => {
```

Comment:
ok to leave like this for now. I have a fix on a separate ticket for this one https://github.com/Jahia/jcontent/pull/1382/commits/cd709cebea186c1cf0b0369137eec4fb737a7b92